samples/ORKCatalog/ORKCatalog/Tasks/TaskListRow.swift

```diff
@@ -503,9 +516,16 @@ enum TaskListRow: Int, CustomStringConvertible {
         let formItem02 = ORKFormItem(identifier: String(Identifier.FormItem02), text: formItem02Text, answerFormat: ORKTimeIntervalAnswerFormat())
         formItem02.placeholder = NSLocalizedString("Your placeholder here", comment: "")
         
+        // An third field for selecting a location.
+        let formItem03Text = NSLocalizedString("Field 03", comment: "")
```

Comment:
ORKCatalog is a demo app, it is ok to not have location question in the form's demo.